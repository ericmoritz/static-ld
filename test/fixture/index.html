<!doctype html>
<html>
  <title>My test blog</title>
  <body>
    <div><a href="./post-1.html">Post 1</a></div>
    <div><a href="./post-2.html">Post 2</a></div>
    <div><a href="./post-3.html">Post 3</a></div>
  </body>
</html>
