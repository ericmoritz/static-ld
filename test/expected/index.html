<!doctype html>
<html>
  <title>My test blog</title>
  <link href="foo.txt" />
  <body>
    
    
  

  
    <div><a href="posts/post-1.html">Post 1</a></div>
  
    <div><a href="posts/post-2.html">Post 2</a></div>
  
    <div><a href="posts/post-3.html">Post 3</a></div>
  


  </body>
</html>